frontend/packages/ceph-storage-plugin/src/components/modals/storage-pool-modal/storage-pool-constant.ts

```diff
@@ -0,0 +1,10 @@
+export const replicas = {
+  ReplicaTWO: '2',
+  ReplicaTHREE: '3',
```

Comment:
There is already one constant defined for replica 3 here - https://github.com/openshift/console/blob/master/frontend/packages/ceph-storage-plugin/src/constants/index.ts#L17

Reply:
ack